01/app.js

```diff
@@ -1 +1,34 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+
+const firstElement = document.querySelector(".comments__item ",".comments__item--newest")
```

Comment:
Tutaj mamy jeden parametr, który jest ciągiem znaków dlatego prawidłowo to `document.querySelector(".comments__item.comments__item--newest")` (bez spacji, przecinak itp.)